schema/360-summary-table-schema.json

```diff
@@ -174,15 +174,15 @@
    "format": "", 
    "title": "Beneficiary Location:Latitude", 
    "required": false, 
-   "type": "string", 
+   "type": "number", 
```

Comment:
should this be ["number", "null"] as with Line 138?

Reply:
No, because JSON table schema doesn't have a null type https://github.com/frictionlessdata/schemas/blob/align-with-current-specs/definitions.json#L27-L42 (any more, it was removed in https://github.com/frictionlessdata/schemas/commit/17917fff8468a5e96c7e6d9eda516cc29f29caf3).